views/homepage.handlebars

```diff
@@ -0,0 +1,22 @@
+<div>
+    <section class="hero">
+        <div class="large-6 columns">
+            <h1>Poke TeamWise</h1>
+        </div>
+        <div class="large-6 columns">
+            <h2>The place where Pokemon fans make battle-ready teams.</h2>
+        </div>
+    </section>
+    <section class="description">
+        <div class="large-6 columns">
+            <h2>Where Great Teams Are Made</h2>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet quisquam inventore quod velit eos consequuntur
+            neque tempore blanditiis atque officia! Reiciendis eum aliquid tempora fugiat et quam, asperiores sit
+            voluptatem?
```

Comment:
Change this text to a description of the web app. What problem(s) does it solve? (For example, I'm really indecisive and can never pick all 6 members). What makes it unique?